system/codesigner/gpgcodesigner.cpp

```diff
@@ -70,9 +83,130 @@ void GpgCodeSigner::initGpg(void)
     if (ret != 0)
         throw makeStringExceptionV(MSGAUD_operator, CODESIGNER_ERR_GPG, "Error running gpg: %s", errmsg.str());
     isGpgV1 = strstr(output.str(), "gpg (GnuPG) 1.");
+    if (createNewGpgHomeDir)
+    {
+        try
+        {
+            Owned<IFile> dir = createIFile(gpgHomeDir);
+            dir->createDirectory();
+        }
+        catch (IException *e)
+        {
+            OERRLOG("Create directory failed: %s", gpgHomeDir.str());
+            e->Release();
+        }
+        createNewGpgHomeDir = false;
+    }
     isInitialized = true;
 }
 
+/**
+ * Import a gpg key
+ *
+ * @param key           The gpg key
+ * @param passphrase    The passphrase for key (optional)
+ *
+ * @return              True if successfully imported
+ *                      False if import failed
+ */
+bool jlib_decl GpgCodeSigner::importKey(const char *key, const char *passphrase)
```

Comment:
trivial jlib_decl not needed.  It is causing the windows build error.

Because the class is not visible from outside the source file there is no need to export any of the functions - same for initForContainer()